doc/website/blog/2023-03-31-preventing-breaking-changes.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: "Preventing breaking changes to your database with Atlas"
+authors: rotemtam
+tags: [ci, linting, devops, dora-metrics]
+---
+
+As applications evolve, database schema changes become inevitable. However, some types
+of changes can break the contract between the backend server and the database, leading
+to errors during deployment and application downtime. Ensuring that these breaking changes
+are detected and prevented is crucial for maintaining system stability and delivering
+a seamless user experience. In this post, we will explore how Atlas can help you prevent
+breaking schema changes from impacting your application.
+
+**What are breaking changes?**
+
+Backward-incompatible changes, also known as breaking changes, are schema changes that
+have the potential to break the contract with applications that rely on the old schema.
+As applications today are expected to be highly-available, which means that we cannot
+shut them down for maintenance, during the deployment phase of a new version we will
+always have old and new code serving traffic side by side. This means, for example, that
+renaming a column from `email_address` to `email` will cause errors applications running
+the previous version of the schema reference the old column name in their queries.
+
+**Manually enforcing backward-compatability**
+
+Preventing breaking changes in production is crucial to maintaining a highly-available
+system. To enforce backward-compatibility most teams rely on two things:
+
+1. **GitOps:** Making sure every change to the database is checked in to
+  source control system, and using automated schema migration tools to apply the change.
+2. **Manual code-review:** Carefully reviewing every schema change to make sure it
+  will not break the contract between the backend and the database.
+
+Checking in database artifacts into source control is a well established technique -
```

Comment:
```suggestion
Checking-in database artifacts into source control is a well established technique -
```
Maybe like this? Reading flow is disrupted (at least for me) if this is not read as one term.